src/promptflow/tests/test_configs/eager_flows/basic_model_config/requirements.txt

```diff
@@ -0,0 +1,2 @@
+--extra-index-url https://azuremlsdktestpypi.azureedge.net/promptflow/
```

Comment:
I remember we have a compliant check that we should not check in such index in code. Let me know if that's still the case

Reply:
Good catch, removed.